templates/fragments/_sca.erb

```diff
@@ -1,70 +1,39 @@
 <%- if @kernel == 'Linux' -%>
-  <%- if @apply_template_os == 'centos' -%>
     <sca>
       <enabled>yes</enabled>
       <scan_on_start>yes</scan_on_start>
       <interval>12h</interval>
       <skip_nfs>yes</skip_nfs>
 
       <policies>
+      <%- if @os['family'] == 'CentOS' -%>
         <policy>cis_rhel7_linux_rcl.yml</policy>
```

Comment:
Only centos 7 is supported?

Reply:
This was WIP, uses the same ruleset as RHEL, it's fixed now.